src/infrastructure/adapters/logging_adapter.py

```diff
@@ -15,28 +15,35 @@ def __init__(self, name: str = "application") -> None:
 
     def debug(self, message: str, *args: Any, **kwargs: Any) -> None:
         """Log debug message."""
+        kwargs.setdefault("stacklevel", 2)
```

Comment:
Isnt there a way to make this addition DRY?

Reply:
fixed